.github/workflows/sast.yml

```
@@ -35,18 +35,20 @@ jobs:
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v2
+      uses: github/codeql-action/init@v3
+      with:
+        languages: ${{matrix.language}}
 
     - name: Setup JDK 17
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v4
       with:
           java-version: 17
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)